src/domains/dashboard/components/head-main.tsx

```diff
@@ -65,13 +66,13 @@ export const HeadMain = ({
             attributes={{
               ...commonAttributes,
               id: "system.io",
-              dimensions: "in",
               chartLibrary: "easypiechart",
               title: "Disk Read",
               width: "11%",
               before: 0,
               unitsCommon: "system.io.mainhead",
               ...(attributesOverrides && attributesOverrides["system.io"]),
+              dimensions: "in",
```

Comment:
if we override the override, the former is no longer an override :) Can we maybe instead tweak the override in the cloud so it doesn't break default values?

Reply:
I thought the same. So, essentially these are hard coded gauge charts which should not be changed by external attribute overrides. 
Meaning if we change any of `aggrMethod`, `units` or `dimensions` (please add anything else if I don't see it here) these Gauges should not be affected correct? So if we change the system.io's dimensions in the cloud the relevant chart should change with a new dimension BUT this gauge should remain constant like it was intended.
So, we could remove attributesOverriddes altogether. I just leave them here in case we want to override something else in the future (I don't maybe this doesn't have any meaning or usefulness and better remove it)

Reply:
@VLegakis what about `chartsAttributes` which you added for correlation mode? Is this information also only applying to dygraph charts? If yes, then maybe we could just remove the attributesOverrides altogether from head-main, etc? Optionally, renaming that param to `dygraphAttributesOverrides`? We have `commonAttributesOverrides` now which we can use to pass params to *every* chart. What do you think?

Reply:
Yes, I think this is the best approach. Metric Correlations is also using only the dygraphs. Removing them then

Reply:
Please take a look @jacekkolasa and let me know